internal/resourcepaths/siloresourcepaths.go

```diff
@@ -0,0 +1,11 @@
+package resourcepaths
```

Comment:
https://github.com/microsoft/hcsshim/pull/1004 You'll need to rebase and then this can be removed